packages/yourloops/components/dashboard-widgets/medical-files/medical-record-list.tsx

```diff
@@ -55,22 +53,22 @@ import { commonComponentStyles } from '../../common'
 import { useAlert } from '../../utils/snackbar'
 import CenteredSpinningLoader from '../../loaders/centered-spinning-loader'
 import { type MedicalRecord } from '../../../lib/medical-files/models/medical-record.model'
+import ListItemButton from '@mui/material/ListItemButton'
+import { type MedicalRecordDialogPayload } from '../models/medical-record-dialog-payload.model'
 
-const useStyle = makeStyles()((theme: Theme) => ({
+const useStyle = makeStyles()(() => ({
   categoryTitle: {
     fontWeight: 600
   },
+  hidden: {
+    visibility: 'hidden'
+  },
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** `visibility: 'hidden'‏` is not a good solution to hide the action button. It can be easily reactivated with devtools and currently a patient can remove a medical record !! Prefer the previous solution with conditional rendering